README.md

```diff
@@ -41,7 +41,7 @@ see `EXT:container_example` for a simple usage of a custom container.
 
 This is an example for create a 2 column container
 
-    \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\B13\Container\Tca\Registry::class)->addContainer(
+    \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\B13\Container\Tca\Registry::class)->registerContainer(
```

Comment:
`registerContainer` was deprecated with https://github.com/b13/container/commit/d5ad1dd3658827ac44ef445b5f94d810bfb63ccd

`addContainer` should be used instead (with changed parameter order)